nominal/core/event.py

```diff
@@ -3,16 +3,17 @@
 import warnings
 from dataclasses import dataclass, field
 from datetime import datetime, timedelta
-from enum import Enum
 from typing import Iterable, Mapping, Protocol, Sequence
 
-from nominal_api import api, event
+from nominal_api import event
 from typing_extensions import Self
 
 from nominal.core import asset as core_asset
 from nominal.core._clientsbunch import HasScoutParams
 from nominal.core._utils.api_tools import HasRid, RefreshableMixin, rid_from_instance_or_string
 from nominal.core._utils.pagination_tools import search_events_paginated
+from nominal.core._utils.query_tools import _create_search_events_query
+from nominal.core.event_types import EventType, SearchEventOriginType
```

Comment:
```suggestion
from nominal.core.event_types import EventType as EventType, SearchEventOriginType as SearchEventoriginType
```

since we moved `EventType`, user code that relies on `nominal.core.event.EventType` will continue to succeed at runtime but their type hinting will fail because it's not explicitly re-exported from event.py.

Reply:
Discussed offline; done!